src/libs/actions/connections/QuickbooksOnline.ts

```diff
@@ -92,6 +90,7 @@ function buildOnyxDataForQuickbooksConfiguration<TSettingName extends keyof Conn
     policyID: string,
     settingName: TSettingName,
     settingValue: Partial<Connections['quickbooksOnline']['config'][TSettingName]>,
+    oldSettingValue?: Partial<Connections['quickbooksOnline']['config'][TSettingName]>,
```

Comment:
Why did we make this value optional with `?`? I think it is not optional and it looks like we are passing it in every call

Reply:
It's because we pass values from QuickbooksOnline config that can be undefined
e.g. [here](https://github.com/Expensify/App/pull/46511/files#diff-abb28f38813d6a787002456e671884b4398b1c92a443a3f62722863252589a98R45)

Reply:
ohh, I see, thanks!